mobile/src/api/customer/customer.ts

```diff
@@ -6,23 +6,59 @@
 import { useQuery } from "@tanstack/react-query";
 import { paymentsApi } from "../../utils/axios";
 import { ICreateCustomer } from "./customer.interface";
+import * as SecureStore from "expo-secure-store";
+import { isNil } from "lodash";
+import { AxiosError } from "axios";
 
 export async function getCustomer(): Promise<any> {
-  // Since API response is inconsistent, we are not able to specify the exact type
-  const response = await paymentsApi.get("/api/customers");
-  return response;
+  try {
+    const response = await paymentsApi.get("/api/customers");
+    return response;
+  } catch (error) {
+    const axiosErrror = error as AxiosError;
+    if (!isNil(axiosErrror?.response?.data)) {
```

Comment:
This seems like this method will always return null. Is this correct? For example if Code is "InvalidStatusError" it will return null and if not. I assume it was suppose to do something with these errors first. Could you change this or completely remove all these if statements as they are doing nothing

Reply:
Fixed. I removed the unused code.